patches/src/main/kotlin/app/revanced/patches/spotify/misc/lyrics/Fingerprints.kt

```diff
@@ -0,0 +1,16 @@
+package app.revanced.patches.spotify.misc.lyrics
+
+import app.revanced.patcher.fingerprint
+
+/**
+ * This method builds and returns an HTTP client for a hardcoded host (declared in this method).
+ */
+internal val httpClientBuilderFingerprint = fingerprint {
```

Comment:
Not sure why multiline is used here

```suggestion
// This method builds and returns an HTTP client with a hardcoded host (declared in this method).
internal val httpClientBuilderFingerprint = fingerprint {
```